src/test/java/uk/ac/ebi/atlas/hcalandingpage/HcaHumanExperimentDaoIT.java

```diff
@@ -33,27 +34,51 @@ void setUp() {
 
     @Test
     void emptyIfNoExperimentsCanBeFound() {
-        assertThat(subject.fetchExperimentAccessions("organism_part", "foobar"))
+        assertThat(subject.fetchExperimentAccessions("organism_part", ImmutableSet.of("foobar")))
                 .isEmpty();
     }
 
     @Test
     void notEmptyForCorrectCharacteristicType() {
-        assertThat(subject.fetchExperimentAccessions("organism_part", "skin"))
+        assertThat(subject.fetchExperimentAccessions("organism_part", ImmutableSet.of("skin")))
                 .isNotEmpty()
                 .containsExactlyInAnyOrder("E-EHCA-2");
     }
 
     @Test
     void notEmptyForWhenValidOnotologyIdIsPassed() {
-        assertThat(subject.fetchExperimentAccessions("organism_part", "UBERON_0000014"))
+        assertThat(subject.fetchExperimentAccessions("organism_part",
+                ImmutableSet.of("http://purl.obolibrary.org/obo/UBERON_0000029")))
```

Comment:
We need to pass just `ontologyId` as this is what we will be getting from `Anatomogram` instead of complete url

Reply:
Got it. Fixed.